src/docs/sdk/performance/dynamic-sampling-context.mdx

```diff
@@ -106,6 +106,10 @@ After the DSC of a particular trace has been frozen, API calls like `set_user` o
 Dynamic Sampling Context is sent to Sentry via the `trace` envelope header and is propagated to downstream SDKs via a baggage header.
 
 All of the values in the payloads below are required (non-optional) in a sense, that when they are known to an SDK at the time a transaction envelope is sent to Sentry, or at the time a baggage header is propagated, they must also be included in said envelope or baggage.
+
+The **only exception** is the `user_id` field.
+To avoid sensitive data being leaked to third parties, `user_id` should only be included, if the <Link to="/sdk/data-handling/#sensitive-data">`send-default-pii`</Link> option was enabled in the init options.
```

Comment:
Let's make it for new DSC - especially since the data here is based on the head SDK.

Reply:
> Only new DSC or also DSC from incoming requests
> Let's make it for new DSC

Yes, I strongly agree with only new DSC. We have to adapt this in all SDKs and hence, eventually, we should not get a user_id in incoming DSC. Also, I don't want to allow exceptions in the DSC immutability for now. 

> Can we also rename this to send_default_pii

Certainly, the only reason why I wrote it like this is because it was already written in that format [in the dev docs](https://develop.sentry.dev/sdk/data-handling/#sensitive-data).

Reply:
I think we're also good for the case that downstream SDKs set different `sendDefaultPii` values than the head-of-trace SDK. Users have to set DS rules on the DSC values from the head SDK anyway, meaning this is the only SDK where the Pii decision should matter

Reply:
See [efbc5a1](https://github.com/getsentry/develop/pull/625/commits/efbc5a15b8f55e8ad150fdd9c0171d0bf832b030) and [8679a0d](https://github.com/getsentry/develop/pull/625/commits/8679a0d0612b97e66309da2010e3a60ea48d745b)